creativecommons/cc-legal-tools-app
# language width is fixed for small devices

## Fixes


- Fixes #418  by @ShivamAhir

## Description
Previously, the language option container was only partially visible on mobile device screens, resulting in a poor user experience. However, with recent modifications, the container is now fully visible on all mobile device screens. This enhancement ensures that users can easily access and interact with the language options without any visibility issues.


## Tests
To ensure the changes are working as expected, follow these steps:

1. Clone the repository locally.
2. Install any necessary dependencies.
3. Run the project locally.
4. Navigate to [http://127.0.0.1:8005/licenses/list.en](http://127.0.0.1:8005/licenses/list.en) in your web browser.
5. Test the layout and functionality across different screen sizes, including small devices.

Verify that the width of language elements adjusts correctly according to the screen size, confirming the applied modifications.


## Screenshots
![image](https://github.com/creativecommons/cc-legal-tools-app/assets/74092725/a271fc63-d959-4a30-a6f7-b985fa46b315)
![image](https://github.com/creativecommons/cc-legal-tools-app/assets/74092725/f5c86876-af84-446d-9090-6dcf71196a43)



## Checklist



  index.md`).
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin


For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
